lib/streamlit/elements/exception_proto.py

```diff
@@ -13,14 +13,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import sys
 import traceback
 
+import streamlit
+from streamlit.errors import StreamlitAPIException
 from streamlit.logger import get_logger
 
 LOGGER = get_logger(__name__)
 
 
+_streamlit_dir = None
```

Comment:
Why can't you initialize this here?

Reply:
Oh, just being overly paranoid about import time. I'll simplify.